docs/quickstart.md

```diff
@@ -85,14 +37,14 @@ the Argo CloudOps service.
 * Start the Argo CloudOps Service (includes vault)
 
     ```sh
-    make ; make up
+    ./scripts/quickstart_run.sh
     ```
 
 * To run in debug mode set log level DEBUG before running
 
     ```
     export ARGO_CLOUDOPS_LOG_LEVEL=DEBUG
-    make ; make up
+    ./scripts/quickstart_run.sh
```

Comment:
I think this should be `bash ./scripts/quickstart_run.sh` to be consistent with other script invocations.